### ISSUE TITLE
Create a page with new designer template not saving content

### ISSUE THREAD

**Issue by fmontes** [dotCMS/core#19982]
**Describe the bug**
When you create a page with a recently created template with the new portlet and add content to it a page with that template, the content is not being saved.

**To Reproduce**
1. Create a new designer template
2. Create a page with that template
3. Add any content
4. Refresh and see the content is gone

**Expected behavior**
It should save the content

**Additional context**
The `uuid` to save the content is going `-1`

![screen_shot_2021-02-22_at_11.53.11_am.png](https://images.zenhubusercontent.com/570fbdcebf93871ccd997222/98755208-5b81-4369-8368-c0607c0dc90c)

I think that's the problem.

**Comment by fmontes** [comment]
The UUID is coming bad from the page api.

<img width="1326" alt="Screen Shot 2021-02-22 at 12 14 38 PM" src="https://user-images.githubusercontent.com/751424/108753035-d85c1180-7509-11eb-8f0b-12787fc2fb95.png">

**Comment by erickgonzalez** [comment]
The issue here is that we're not sending the uuid in the payload, so it's using the default value instead. We need to generate and set the right uuid, to accomplish this we need to check how many same containers the template has, will behave something like this:
```
Template 
     containerA, 1
     containerB, 1
     containerC, 1
     containerA, 2
```

**This is only for design templates**

**Comment by erickgonzalez** [comment]
PR: https://github.com/dotCMS/core/pull/19995

**Comment by dsilvam** [comment]
@fmontes label this as `21.02.2` instead of `21.03`

**Comment by bryanboza** [comment]
Tested on release-21.02.2 and now I'm unable to reproduce.

**Comment by erickgonzalez** [comment]
PR tests: https://github.com/dotCMS/core/pull/19997

**Comment by bryanboza** [comment]
Fixed, tested on release-21.03 // Postgres // FF
